pkg/sdn/plugin/node.go

```diff
@@ -248,12 +252,9 @@ func (node *OsdnNode) Start() error {
 			return true, nil
 		})
 
-	log.V(5).Infof("Creating and initializing openshift-sdn pod manager")
-	node.podManager, err = newPodManager(node.host, node.localSubnetCIDR, node.networkInfo, node.kClient, node.policy, node.mtu, node.oc)
-	if err != nil {
-		return err
-	}
-	if err := node.podManager.Start(cniserver.CNIServerSocketPath); err != nil {
+	// Kubelet has initialized, now we have a valid node.host
```

Comment:
Oops, ok, so then the comment above about "all OsdnNode fields have been initialized" is a lie then. Fix that in some way. Then LGTM.